target/snitch_cluster/Makefile

```
@@ -35,10 +35,37 @@ CHISEL_PATH := $(ROOT)/hw/chisel/
 
 include $(ROOT)/target/common/common.mk
 
+############
+# Programs #
+############
+
+REGGEN          ?= $(shell $(BENDER) path register_interface)/vendor/lowrisc_opentitan/util/regtool.py
+CLUSTER_GEN     ?= $(ROOT)/util/clustergen.py
+CLUSTER_GEN_SRC ?= $(wildcard $(ROOT)/util/clustergen/*.py)
+WRAPPER_GEN     ?= $(ROOT)/util/wrappergen/wrappergen.py
+
+#########################
+# Files and directories #
+#########################
+
+SW_DIR             = sw
+TARGET_C_HDRS_DIR ?= $(SW_DIR)/runtime/common
+GENERATED_DIR     ?= $(MKFILE_DIR)generated
+
 ####################
 # SNAX Generations #
 ####################
 
+define generate_snax_gen
+SNAX_GEN := \
+	$(GENERATED_DIR)/$(1)/$(1)_csrman_CsrManager.sv \
+	$(GENERATED_DIR)/$(1)/$(1)_streamer_StreamerTop.sv \
+	$(GENERATED_DIR)/$(1)/$(1)_csrman_wrapper.sv \
+	$(GENERATED_DIR)/$(1)/$(1)_streamer_wrapper.sv \
+	$(GENERATED_DIR)/$(1)/$(1)_wrapper.sv
+endef
```

If you think about it, I would also add the `-t` also inside to reduce duplicates. So...

```suggestion
define generate_snax_gen
SNAX_GEN := \
	$(GENERATED_DIR)/$(1)/$(1)_csrman_CsrManager.sv \
	$(GENERATED_DIR)/$(1)/$(1)_streamer_StreamerTop.sv \
	$(GENERATED_DIR)/$(1)/$(1)_csrman_wrapper.sv \
	$(GENERATED_DIR)/$(1)/$(1)_streamer_wrapper.sv \
	$(GENERATED_DIR)/$(1)/$(1)_wrapper.sv
	
	VSIM_BENDER += -t $(1)
	VLT_BENDER  += -t $(1)
	VCS_BENDER  += -t $(1)
	
endef
```

And you know about it where to adjust it.

---

I think you're right, although I leave that up to @jorendumoulin  in #112